### DISCUSSION TITLE
How can I get built wheel after "pip install -v -U git+https://github.com/facebookresearch/xformers.git@main#egg=xformers"

### DISCUSSION THREAD

**Discussion by terrificdm** [facebookresearch/xformers#637] [Q&A]
I saw below information after I ran pip install, so my question is, is there any way to get this built file for another installation? Because after success of pip install I cannot find this wheel file. Thanks.
```
Created wheel for xformers: filename=xformers-0.0.15+7aea476.d20230112-cp310-cp310-linux_x86_64.whl size=7469735 sha256=bd0d81c882cdd65aac8083bf79371053f0aa8ffaec3a57a432ccd3a551fd9249
  Stored in directory: /tmp/pip-ephem-wheel-cache-xpdr47u_/wheels/c6/e1/85/5f7882fdae87b53eb35001eb734a2606aa0fb5475446543684
```

**Comment by danthe3rd** [comment]
Hi,
Something like "`python setup.py bdist_wheel`" should do the trick, but it's more of a question for pip/setuptools rather than xformers

**Reply by terrificdm** [reply]
I see, thanks for your help.
